### PR TITLE
Update smui Dockerfile

### PR DESCRIPTION
**Pull request by Paul-Blanchaert**
Apparenlty 1024 is uid for smui user

### CODE REVIEW

**Comment by epugh** [comment]
Can you share a bit more on why 1024 is the uid for smui user?  where do we see that set?

File: smui/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,10 @@
 FROM querqy/smui:3.14.0
 
-COPY conf/smui2solrcloud.sh /smui/conf/smui2solrcloud.sh
-COPY conf/*.py /smui/conf/
-COPY conf/predefined_tags.json /smui/conf/predefined_tags.json
+COPY --chown=1024:1024 conf/* conf/
```
Comment (epugh) [inline smui/Dockerfile:3]: I notice below a `USER smui`, could this line be `--chown=smui:smui conf/* conf/`????

**Comment by Paul-Blanchaert** [comment]
> Can you share a bit more on why 1024 is the uid for smui user? where do we see that set?
Please see [querqy/smui](https://github.com/querqy/smui/blob/f692a2f2638c07f3b264631c3ab52284069e0118/Dockerfile#L42)

File: smui/Dockerfile (hunk 1 above)
Comment (Paul-Blanchaert) [reply smui/Dockerfile:3]: > I notice below a `USER smui`, could this line be `--chown=smui:smui conf/* conf/`????

probably; should be the same... I'll check and send a commit...

**Comment by epugh** [comment]
Thanks, this is much cleaner....

**Comment by epugh** [comment]
Should the same change be applied to https://github.com/querqy/chorus-elasticsearch-edition as well?

**Comment by Paul-Blanchaert** [comment]
I believe it can be the same


------------------------------


Kind regards,

Paul Blanchaert <https://www.search-solutions.net/p/paul>

www.search-solutions.net

Tel: +32 497 05.01.03

[image: View my profile on LinkedIn]
<http://be.linkedin.com/in/paulblanchaert>view
<http://be.linkedin.com/in/paulblanchaert> or connect
<https://www.linkedin.com/inviteFromProfile?from=profile&key=682342&firstName=Paul&lastName=Blanchaert>

------------------------------

Please consider the environment before printing this e-mail.

This message is explicitly subject to the conditions of the e-mail
disclaimer, available via the following link: mail-disclaimer
<https://www.search-solutions.net/mail-disclaimer>. If you are unable to
consult this e-mail disclaimer, please notify the sender at once.

**Review by mkr — APPROVED** [review]

**Comment by epugh** [comment]
just retrested and looks good..  but, for bonus points, goign to bump SMUI...

**Pull request merged**